aries-site/.env.local

```diff
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_FEEDBACK_APP_API_TOKEN=acLCL6dZED3y8nZhqPSq1R6rCTFh1adqrYEXPdfY
```

Comment:
Can we add a comment referencing the Next docs on this? https://nextjs.org/docs/basic-features/environment-variables